app/models/cms/section.rb

```diff
@@ -49,11 +49,26 @@ def to_xml(options = {})
         xml.created_at created_at.xmlschema
         xml.updated_at updated_at.xmlschema
       end
-      x.partial_path partial_path
-      x.public public
       x.title title
-      x.parent_id parent_id
       x.system_name system_name
+      x.public !public.nil?
+      if options[:short]
+        x.parent_id parent_id
+      else
+        x.parent_ do |p|
```

Comment:
Is this `parent_` defined by Nokogiri? Couldn't find it in the docs 😅

Reply:
The same case as above, I want to create a `<parent>` tag, but `#parent` is defined in the builder ([Docs](https://msp-greg.github.io/nokogiri/Nokogiri/XML/Builder.html#parent-instance_method))